batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentListController.java

```diff
@@ -0,0 +1,117 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Student;
+import com.lftechnology.batch7crud.service.StudentServices;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by leapfrog on 1/18/16.
+ */
+@WebServlet(name = "StudentListController", urlPatterns = {"/Students/*"})
+public class StudentListController extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String urlpath = request.getPathInfo();
+        if(urlpath == null)
+            list(request,response,1);
+        else {
+            String[] urlparts = urlpath.split("/");
+            if("NewEntry".equals(urlparts[1])){
+                createProcess(request, response);
+            }
+            else if("edit".equals(urlparts[2])){
+                editProcess(request, response, Integer.parseInt(urlparts[1]));
+            }
+            else if("delete".equals(urlparts[2])){
+                deleteProcess(request,response,Integer.parseInt(urlparts[1]));
+            }
+        }
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)